Assets/Scripts/EncounterScripts/Encounter.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+public class Encounter : MonoBehaviour 
+{
+	public string id;
```

Comment:
id is never used, and also should ideally be stored as something other than a string (if we need it...? unsure of its purpose)